packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummary.tsx

```diff
@@ -146,7 +148,36 @@ export function BridgeReviewSummary() {
 
   const toNetwork = useMemo(() => to?.network, [to]);
 
-  const fetchGasEstimate = useCallback(async () => {
+  const fetchTransferGasEstimate = useCallback(async () => {
+    if (!tokenBridge || !amount || !from || !to || !token) return;
+
+    const tokenToTransfer = token?.address?.toLowerCase() ?? NATIVE.toUpperCase();
+    const gasEstimateResult = {
+      gasEstimateType: GasEstimateType.BRIDGE_TO_L2,
+      fees: {},
+      token: checkout.config.networkMap.get(from!.network)?.nativeCurrency,
+    } as GasEstimateBridgeToL2Result;
+    let estimatePromise: Promise<BigNumber>;
+    if (tokenToTransfer === NATIVE.toLowerCase()) {
+      estimatePromise = from.web3Provider.estimateGas({
+        to: toAddress,
+        value: utils.parseUnits(amount, token.decimals),
+      });
+    } else {
+      const erc20 = getErc20Contract(tokenToTransfer, from.web3Provider.getSigner());
+      estimatePromise = erc20.estimateGas.transfer(toAddress, utils.parseUnits(amount, token.decimals));
+    }
+    const [estimate, gasPrice] = await Promise.all([estimatePromise, from.web3Provider.getGasPrice()]);
```

Comment:
@dom-murray I got stuck on the confirmation screen - I think it's failing on this line silently.

![image](https://github.com/immutable/ts-immutable-sdk/assets/13159042/b0f4157e-9096-4837-819b-bf7d467b9eff)